CHANGELOG.md

```diff
@@ -29,6 +29,9 @@
 - Upgrade the Helm dependency version from `v3.0.1` to `v3.0.2`. ([#2621](https://github.com/operator-framework/operator-sdk/pull/2621)) 
 - Changed the scaffolded `serveCRMetrics` to use the namespaces informed in the environment variable `WATCH_NAMESPACE` in the MultiNamespace scenario. ([#2603](https://github.com/operator-framework/operator-sdk/pull/2603)) 
 - Improve skip metrics logs when running the operator locally in order to make clear the information for Helm based operators. ([#2603](https://github.com/operator-framework/operator-sdk/pull/2603))
+- Breaking Change: Renamed --namespace flag as --watch-namespace in operator-sdk run local command. (#2617)
+- Breaking Change: Split --namespace flag into --operator-namespace and --watch-namespace in operator-sdk test local command to decouple the operator namespace from the watch namespace. (#2617)
+- Breaking Change: Split `ctx.GetNamespace()` in `pkg/test` into `ctx.GetOperatorNamespace()` and `ctx.GetWatchNamespace()`. (#2617) 
 
```

Comment:
In the "Deprecated" section:

```suggestion
- **Breaking Change:** The `--namespace` flag from `operator-sdk run --local` and `operator-sdk test --local` command was deprecated and will be removed in the future versions. Use `--watch-namespace` and `--operator-namespace`  instead of. ([#2617](https://github.com/operator-framework/operator-sdk/pull/2617))
- **Breaking Change:** The method `ctx.GetNamespace()` from the `pkg/test` is deprecated and will be removed in the future versions. Use `ctx.GetOperatorNamespace()` and `ctx.GetWatchNamespace()` instead of. ([#2617](https://github.com/operator-framework/operator-sdk/pull/2617))
```

In the "Added" section:

``
- The flag `--watch-namespace` and `--operator-namespace` was added to `operator-sdk run --local` and `operator-sdk test --local` commands in order to replace the flag `--namespace` which was  deprecated.([#2617](https://github.com/operator-framework/operator-sdk/pull/2617))
- The methods `ctx.GetOperatorNamespace()` and `ctx.GetWatchNamespace()` was added `pkg/test` in order to replace `ctx.GetNamespace()` which is  deprecated. ([#2617](https://github.com/operator-framework/operator-sdk/pull/2617))
```

NOTES:
- Use the format for `Breaking Change:` that has been used `**Breaking Change:**`
- Use the format to add the pr which has been used
- We should deprecate the current flags and methods instead of remove.